internal/ingress/controller/store/store.go

```diff
@@ -659,6 +662,10 @@ func (s *k8sStore) syncIngress(ing *networkingv1beta1.Ingress) {
 			if path.Path == "" {
 				copyIng.Spec.Rules[ri].HTTP.Paths[pi].Path = "/"
 			}
+
+			if path.PathType == nil || *path.PathType == networkingv1beta1.PathTypeImplementationSpecific {
```

Comment:
> *path.PathType == networkingv1beta1.PathTypeImplementationSpecific

I'd put this in a separate if condition with a comment saying we currently do not support implementation specific path type and ignore it

Reply:
> and ignore it

The default type, when `PathType` is not defined is `ImplementationSpecific`, something makes no sense in ingress-nginx. The default we have (previous to these new fields) is PathPrefix. That's the reason. I changed the code to make this clear

Reply:
> I changed the code to make this clear

👍 looks great now!